src/javascript/app_2/App/Components/Form/DropDown/items.jsx

```diff
@@ -12,27 +12,35 @@ const Items = ({
     value,
 }) => (
     items.map((item, idx) => (
-        <React.Fragment key={idx}>
-            <div
-                className={classNames('list__item', {
-                    'list__item--selected': value === item.value,
-                })}
-                key={idx}
-                name={name}
-                value={item.value}
-                onClick={handleSelect.bind(null, item)}
-            >
-                {!!has_symbol && item.has_tooltip &&
-                <Tooltip alignment='top' className='list__item-tooltip' message={getCurrencyName(item.value)}>
-                    <i><span className={`symbols list__item-tooltip-symbols symbols--${(item.text || '').toLowerCase()}`} /></i>
-                </Tooltip>
-                }
-                {!!has_symbol && !item.has_tooltip &&
+        <div
+            className={classNames(
+                'list__item',
+                { 'list__item--selected': value === item.value }
+            )}
+            name={name}
+            value={item.value}
+            onClick={handleSelect.bind(null, item)}
+            key={idx}
+        >
+            {!!has_symbol && item.has_tooltip &&
+                <Tooltip2
+                    alignment='left'
+                    message={getCurrencyName(item.value)}
+                >
+                    <span
+                        className={`symbols list__item-symbol symbols--${(item.text || '').toLowerCase()}`}
```

Comment:
please use `classNames` to segregate the `classNames` and check if `item.text` exists before appending `className`, because in the event that `item.text` doesn't exist, it will pass `symbols-- ` as a className.

Reply:
done